packages/shared/src/index.ts

```diff
@@ -9,3 +9,4 @@ export type {
   WebAppCommand,
 } from "./types/native";
 export { parseActiveTab, trimFullPath } from "./logic/navigation";
+export * from "./client";
```

Comment:
I don't think we should export the client types at root, since they'd conflict if we were to try to export both client + urbit types at root, and they should be symmetrical. But not critical this second. Maybe just add a todo there?

Reply:
Was thinking that as well. Let's just export them as a namespace for now.